pkg/volume/util/hostutil/hostutil_windows.go

```diff
@@ -143,6 +134,9 @@ func (hu *HostUtil) GetMode(pathname string) (os.FileMode, error) {
 	if err != nil {
 		return 0, err
 	}
+	if info.Mode()&os.ModeIrregular != 0 {
```

Comment:
this seems like a really low-level method... is ModeIrregular really an error, or is that a signal callers of this function could be looking for?

Reply:
Recently ModeIrregular was updated so non-symlink reparse points are treated as such.
https://github.com/golang/go/issues/61893

This is a good question tho, i need to do a little bit more research

Reply:
I couldn't find the caller of this function, so it may be okay to drop this change.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_pods.go/#L345, right?